modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service-power-z.adoc

```diff
@@ -0,0 +1,112 @@
+[id="sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service-ibm-power-z_{context}"]
+= Connecting the Spring PetClinic sample application to the PostgreSQL database service
+
+To connect the sample application to the database service, you must create a `ServiceBinding` custom resource (CR) that triggers the {servicebinding-title} to project the binding data into the application.
+
+[discrete]
+.Procedure
+
+. Create a `ServiceBinding` CR to project the binding data:
++
+[source,terminal]
+----
+$ oc apply -f - << EOD
+---
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+    namespace: my-postgresql
+spec:
+    services: <1>
+    - group: postgresql.dev4devs.com
+      version: v1beta1
+      kind: Database <2>
+      name: sampledatabase
+      version: v1alpha1
+    application: <3>
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+EOD
+----
+<1> Specifies a list of service resources.
+<2> The CR of the database.
+<3> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec.
++
+The output verifies that the `ServiceBinding` CR is created to project the binding data into the sample application.
++
+[source,terminal]
+.Example output
+----
+servicebinding.binding.operators.coreos.com/spring-petclinic-rest created
+----
+
+. To verify that the binding is successful, check the status conditions of the binding resource:
++
+----
+$ oc get servicebindings spring-petclinic-rest -n my-postgresql -o jsonpath-as-json='{.status.conditions}'
+----
++
+[source,terminal]
+.Example output
+----
+[
+  [
+    {
+      "lastTransitionTime": "2021-09-06T13:42:28Z",
+      "message": "",
+      "reason": "DataCollected",
+      "status": "True",
+      "type": "CollectionReady",
+    },
+    {
+      "lastTransitionTime": "2021-09-06T13:42:28Z",
+      "message": "",
+      "reason": "ApplicationUpdated",
+      "status": "True",
+      "type": "InjectionReady",
+    },
+    {
+      "lastTransitionTime": "2021-09-06T13:42:28Z",
+      "message": "",
+      "reason": "ApplicationsBound",
+      "status": "True",
+      "type": "Ready",
+    },
+  ],
+]
+----
++
+By default, the values from the binding data of the database service are projected as files into the workload container that runs the sample application.
+
+. Once this is created, you can go to the topology to see the visual connection.
++
+.Connecting spring-petclinic-rest to sampledatabase
+image::img3_power.png[]
```

Comment:
Since we pulled the operator deployment out of the my-postgresql namespace this is the actual topology

<img width="581" alt="image" src="https://user-images.githubusercontent.com/8323212/144617478-903ac7ba-1aa9-45b4-afc9-dfba21b7e6c4.png">